cryptostock_api/cryptostock/notification/models.py

```diff
@@ -160,7 +202,31 @@ def render(data: Dict[str, Any], *args) -> Any:
 
 
 @TemplaterRegister.register(
-    notification_type=NotificationType.SUCCESS_OFFER, consumer_type=ConsumerType.EMAIL
+    notification_type=BrokerNotificationType.ASSET_PRICE_HAS_BEEN_RAISED_ON_MARKET
+)
+@TemplaterRegister.register(
+    notification_type=BrokerNotificationType.ASSET_PRICE_HAS_BEEN_DROPPED_ON_MARKET
+)
+@TemplaterRegister.register(
+    notification_type=BrokerNotificationType.ASSET_APPEARED_ON_MARKET
+)
```

Comment:
Can we improve TemplaterRegister to support usage like
```suggestion
@TemplaterRegister.register(
    notification_types=[
        BrokerNotificationType.ASSET_APPEARED_ON_MARKET,
        BrokerNotificationType.ASSET_PRICE_HAS_BEEN_DROPPED_ON_MARKET,
        BrokerNotificationType.ASSET_PRICE_HAS_BEEN_RAISED_ON_MARKET,
    ]
)
```

Reply:
I suppose, yes. It is simple good idea

Reply:
but, It is not so easy, as I thought for base logic Templater. If we will add logic for list of types, our class will became less universal